css/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+    background: #FFF;
+}
+
+.page-header {
+    background: #FFF;
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+}
+
+.header-icon {
+    width: 151px;
+    height: 101px;
+    flex-shrink: 0;
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+.navigation {
+    display: flex;
+    align-items: center;
+}
+
+.nav-list {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+}
```

Comment:
you must cancel default settings of browser, here your list gas a default padding-left and margin top and bottom

Reply:
<img width="575" alt="image" src="https://github.com/artem-palytsia/github-hotel_project/assets/115567693/6d9efcc8-fc12-4b6f-a2bc-81d66aae5863">